conda-forge.yml

```
@@ -6,4 +6,7 @@ conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
+provider:
+  linux_aarch64: default
+  linux_ppc64le: default
```

Maybe use azure emulated build will work, maturin-feedstock uses it.

```suggestion
  linux_ppc64le: azure
```